src/lib/hooks.js

```diff
@@ -11,7 +11,7 @@ import Triggers from './triggers';
 const createOrUpdateGuest = async (guest) => {
 	const { token } = guest;
 	token && await store.setState({ token });
-	const user = await Livechat.grantVisitor({ visitor: { ...guest } });
+	const user = await Livechat.grantVisitor({ visitor: { ...guest } }).catch(() => null);
```

Comment:
Wouldn't this ignore any error thrown by the API? Feel we should at least log it.

Reply:
One more suggestion: instead of .catch() pipeline, I'd recommend using try/catch block since it not only improves code readability but also preserves the parent call context so you won't have to use .bind() method

Reply:
I'd also prefer to go with the try catch as it's more explicit 👀